src/Chapter2.hs

```diff
@@ -620,7 +628,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven (x:_:xs) = x : takeEven xs
+takeEven [x] = x : []
```

Comment:
Great pattern matching skills 💯 

Also, `[x]` is the same as `x : []` 🙂

Reply:
Thanks. Is one of them preferred?

Reply:
Usually, `[x]` is used. Though it is desugared to `x : []` as you wrote 🙂